src/test/java/org/isf/visits/test/TestVisit.java

```diff
@@ -35,7 +35,7 @@
 	private GregorianCalendar date = new GregorianCalendar(10, 9, 8);
 	private String note = "TestNote";
 	private boolean sms = true;
-	private String duration = "10";
+	private Integer duration = new Integer(10);
```

Comment:
This is unnecessary boxing; just use `private Integer duration = 10;`

Reply:
Right!